src/components/Tabs.tsx

```diff
@@ -34,6 +34,8 @@ export interface Tab<V extends string = string> {
 
 type TabsContentXss = Xss<Margin>;
 
+export type TabType = "default" | "lineTab";
```

Comment:
Thought: Maybe include an in-source comment about what `lineTab` is? Afaict it's just a thicker line on the bottom?

Also naive question, but if someone on UX is asking for this b/c "they like this tab style better", should we just change all of our tabs to be this new style? Like why we keep the old one we don't like anymore?

(Disclaimer these questions to UX usually go like 50/50, but :crossed_fingers: they're fine with "ah sure, new tabs everywhere!")

If they do want to keep both styles, I'd really like (cc @bdow and @jeffreyqua ) to work into our component library documentation the rationale for "when you use default tabs" vs. "when to use line tabs", as I'd like to think there is some sort of rule-of-thumb driving this change more than just "I think line tabs look better".

Reply:
I agree with this comment. I believe the only reason why we are keeping both around for now is because we aren't sure how the new style will look on existing pages. @jeffreyqua can you confirm the reasoning for needing two styles?

Reply:
Okay just walked through it with Jeff Qua after reviewing the current uses of Tabs and we're good to make this change a style replacement. Just pushed up those changes for a new review

Reply:
That's amazing @blambillotte (and @jeffreyqua !), thanks for taking the extra time to do this!